core/src/test/kotlin/com/exactpro/th2/read/db/core/impl/DefaultValueTransformerTest.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.read.db.core.impl
+
+import com.exactpro.th2.read.db.core.impl.DefaultValueTransformer.Companion.INSTANCE
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments.of
+import org.junit.jupiter.params.provider.MethodSource
+import java.math.BigDecimal
+import java.sql.Date
+import java.sql.Time
+import java.sql.Timestamp
+import java.time.Instant
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.ZoneId
+import kotlin.test.assertEquals
+
+class DefaultValueTransformerTest {
+    @ParameterizedTest
+    @MethodSource("values")
+    fun `transform test`(value: Any, expected: String) {
+        assertEquals(expected, INSTANCE.transform(value), "value: $value, class: ${value::class.java}")
+    }
+
+    companion object {
+        @JvmStatic
+        fun values() = listOf(
```

Comment:
note: Instead of having this method in the companion object you could move it to the class itself and use `@TestInstance(PER_CLASS)` annotation on the test class